listeners/websocket.js

```diff
@@ -0,0 +1,31 @@
+var WebSocketClient = require('websocket').client;
+var deviceInfo = require('./../data/device');
+
+var init = exports.init = function(conf, logger) {
+    var client = new WebSocketClient();
+
+    client.on('connectFailed', function() {
+        logger.error("Websocket cannot connect.");
+        setTimeout(function() {
+            init(conf, logger);
+        }, parseInt(conf.connector.ws.retryTime));
+    });
+    client.on('connect', function(connection){
+        logger.info('Websocket listener started on port: ' + conf.connector.ws.port);
+        var initMessageObject = {
+            "type": "device",
+            "deviceId": deviceInfo.device_id,
+            "deviceToken": deviceInfo.device_token
+        };
+        connection.sendUTF(JSON.stringify(initMessageObject));
+        connection.on('close', function() {
+            logger.info("Websocket connection closed.");
+            init(conf, logger);
+        });
+        connection.on('message', function(message) {
+            logger.info('Fired STATUS: ', message.utf8Data);
+        });
+    });
+    client.connect('ws://' + conf.connector.ws.host + ':' + conf.connector.ws.port, 'echo-protocol');
```

Comment:
It would be nice to give the user an option to choose between ws and wss. If so, then you should include protocol in message I mentioned above